test/dotnet.Tests/PackagedCommandTests.cs

```diff
@@ -65,19 +65,19 @@ public void TestProjectToolIsAvailableThroughDriver(string appName, bool useCurr
             var testInstance = TestAssets.Get(appName)
                 .CreateInstance()
                 .WithSourceFiles()
-                .WithRestoreFiles();
+                .WithNuGetConfig(new RepoDirectoriesProvider().TestPackages);
```

Comment:
I'm not sure why your original change (Prepare.targets) needs these changes. They seem unrelated.